easybuild/framework/easyblock.py

```diff
@@ -1060,7 +1060,9 @@ def make_module_req(self):
                 raise EasyBuildError("Failed to change to %s: %s", self.installdir, err)
 
             lines.append('\n')
+            self.dry_run_msg("List of paths that would be searched and added to modulefile:")
             for key in sorted(requirements):
+                self.dry_run_msg("\t'%s': %s" % (key, requirements[key] ))
```

Comment:
add another `self.dry_run_msg('')` to get an empty line between this and the next block in the dry run output